src/v2/guide/list.md

```diff
@@ -211,17 +211,17 @@ new Vue({ el: '#range' })
 </script>
 {% endraw %}
 
-### Components and `v-for`
+### Composants et `v-for`
 
-> This section assumes knowledge of [Components](components.html). Feel free to skip it and come back later.
+> Cette partie suppose que vous ayez pris connaissance du contenu de [Composants](components.html). Vous pouvez la passer pour le moment et y revenir plus tard.
 
-You can directly use `v-for` on a custom component, like any normal element:
+Vous pouvez directement utiliser `v-for` sur un composant personnalisé, comme sur n'importe quel autre élément standard :
 
 ``` html
 <my-component v-for="item in items"></my-component>
 ```
 
-However, this won't automatically pass any data to the component, because components have isolated scopes of their own. In order to pass the iterated data into the component, we should also use props:
+Cependant, cela ne passera pas automatiquement les données au composant, et cela car les composants ont leurs propres portées isolées. Dans l'optique de passer les données itérées au composant, nous devons utiliser les props conjointement :
```

Comment:
Pour « nous devons aussi utiliser les props » j'ai un soucis car pour moi le mot aussi signifie plutôt « too » dans le sens ou : « On a déjà passé les props pour faire un truc, et donc pour itérées sur les composants on doit **aussi** les utilisers ».

Là c'est pas le cas, là c'est *also* dans le sens « on doit utiliser ce truc en plus » donc j'avais pensé à ça 

> nous devons utiliser les props en plus

Mais bof bof. Je vais mettre « en plus » car c'est vrai que conjointement est éloigné de l'original même si c'est exactement ce que ça veut dire.